manim/mobject/types/vectorized_mobject.py

```diff
@@ -843,6 +844,51 @@ def get_subcurve(self, a, b):
         vmob.pointwise_become_partial(self, a, b)
         return vmob
 
+    """
+    The next 3 methods are for getting or changing the direction of
+    points in 2D. The direction can either be "CCW", counterclockwise,
+    or "CW", clockwise. The default is counterclockwise. This also
+    determines how the objects are drawn which can by changed by
+    reverse_direction or force_direction. This can also be used to make
+    operations on objects that are dependent on direction much easier.
+    """
```

Comment:
this likely won't get printed anywhere; consider either adding to each mobject's docstring or to VMobject